### PR TITLE
Correct grammatical errors

### PR DESCRIPTION
**Pull request by DaniGerr**
Also add a return message for ``if (room.id !== 'wifi') return false;``.

### CODE REVIEW

**Comment by DanielCranham** [comment]
Theres no need for a return message under if (room.id !== 'wifi') return false; because of the namespace for the command, it will return that the return message you've suggested without adding those return messages in.

File: chat-plugins/wifi.js [hunk 1]
```diff
@@ -383,24 +383,24 @@ var commands = {
...
-			'- question <small>or</small> qg <em>User, Prize, Question, Answer</em> - Start a new question giveaway (Requires: % @ #).<br />' +
-			'- lottery <small>or</small> lg <em>User, Prize, Number of Winners (optional)</em> - Starts a lotto giveaway. The number of winners is set to 1 by default. (Requires: % @ # & ~)<br />' +
-			'- changequestion - Changes the question of a question giveaway. Can only be done before the giveaway starts. (Requires: % @ #)<br />' +
+			'- question <small>or</small> qg <em>User, Prize, Question, Answer</em> - Start a new question giveaway. Requires: % @ #<br />' +
+			'- lottery <small>or</small> lg <em>User, Prize, Number of Winners (optional)</em> - Starts a lotto giveaway. The number of winners is set to 1 by default. Requires: % @ # & ~<br />' +
+			'- changequestion - Changes the question of a question giveaway. Can only be done before the giveaway starts. Requires: % @ #<br />' +
 			'- changeanswer - Changes the answer of a question giveaway. (Requires: % @ #)<br />' +
-			'- end - Forcibly ends the current giveaway (Requires: % @ #)<br />'
+			'- end - Forcibly ends the current giveaway. Requires: % @ #<br />'
```
Comment (TheImmortal) [inline chat-plugins/wifi.js]: This is on purpose. All the help commands display the required ranks in this format. If you are proposing changing it, change it everywhere and in its own pull request.

File: chat-plugins/wifi.js [hunk 2]
```diff
@@ -249,7 +249,7 @@ var commands = {
 	quiz: 'qg',
 	question: 'qg',
 	qg: function (target, room, user) {
-		if (room.id !== 'wifi') return;
+		if (room.id !== 'wifi') return this.sendReply('This command can only be used in the Wi-Fi room.');
```
Comment (codelegend) [inline chat-plugins/wifi.js]: I don't think there is a need to send the error message for staff commands.

**Comment by codelegend** [comment]
Fixed in PR #1411 .

**Pull request closed without merging**